test/y2storage/md_test.rb

```diff
@@ -217,4 +217,9 @@
 
   end
 
+  describe "#software_defined?" do
```

Comment:
I know the corresponding tests are in `md_member_test.rb` and `md_container_test.rb`, but I think it would be better to understand the method if we would have here a more rich example with some objects returning true and others returning false.